packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -116,9 +129,28 @@ export const Webchat = forwardRef((props, ref) => {
     updateTheme({ ...webchatState.theme, ...props.theme })
   }, [props.theme])
 
+  const openMenu = () => {
```

Comment:
if the method also closes the menu , maybe we could call it toggleMenu instead of openMenu?